resources/generate_book.py

```diff
@@ -515,7 +515,13 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
             # todo: supports, more on collapses?
         )),
         entry('prospecting', 'Prospecting', 'tfc:metal/propick/wrought_iron', pages=(
-            image('tfc:textures/gui/book/wip.png'),
+            text('You remembered where you picked up those $(l:getting_started/finding_ores)small metal nuggets$(), right? Finding additional ores may require extensive exploration and mining. You should become very familiar with the $(l:the_world/ores_and_minerals)Ores and Minerals$(). If you need a specific resource, you must find the rock type it spawns in, either under your feet or across the world.'),
+            # todo: reference configuration values rather than defaults?
+            text('When picking up small nuggets or mining exposed ore becomes unsatisfying, it is time to consider prospecting to find concealed ore veins.$(br2)The small nuggets occur when ore is nearby, within 15 blocks horizontally and 35 vertically.$(br2)If you find the center of a group of nuggets, it\'s likely that the vein is beneath you.'),
+            item_spotlight('tfc:metal/propick/copper', 'Prospector\'s Pick', text_contents='If you\'re looking for minerals, which have no metal nuggets, or you can\'t find the vein by guessing, it\'s time to pull out the $(thing)Prospector\'s Pick$(). It searches the 25x25x25 area centered on the block clicked, and reports to the action bar the amount and type of ore located.'),
+            text('The Prospector\'s Pick can give false negatives, but will never give a false positive. Higher tier tools can reduce or eliminate false negatives.$(br2)A propick of the same metal tier will always have the same result when used on the same block unless ores were removed.$(br2)If the Prospector\'s Pick finds multiple ores nearby, it will select only one to report.'),
```

Comment:
I don't know if its obvious what false negative means in this case to someone who hasn't played. I would say more like, "there is a chance it will not find anything, even if there is something there" or something like that